client/src/common/MomentumClient.js

```diff
@@ -4,8 +4,18 @@ const getCourses = () => {
     return ApiClient(`course`);
 };
 
+const createCourse = (course) => {
+    return ApiClient('course', { body: course });
+}
```

Comment:
Could you add the method "POST" to this please?

Reply:
I didn't add before it because the code defaults to POST with a body.

But I added it for clarity.